src/components/TradesWidget/index.tsx

```diff
@@ -1,31 +1,71 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 
 import { ContentPage } from 'components/Layout/PageWrapper'
 import { CardTable } from 'components/Layout/Card'
 
 import { useWalletConnection } from 'hooks/useWalletConnection'
 import { useTrades } from 'hooks/useTrades'
 
-import { TradeRow } from 'components/TradesWidget/TradeRow'
+import { TradeRow, classifyTrade } from 'components/TradesWidget/TradeRow'
+
+import { formatPrice, formatSmart } from '@gnosis.pm/dex-js'
+
+import { Trade } from 'api/exchange/ExchangeApi'
+
+import { displayTokenSymbolOrLink } from 'utils/display'
+import { FileDownloaderLink } from 'components/FileDownloaderLink'
+
+const formatTradeAsCSV = (trade: Trade, now: Date): string => {
+  return [
+    displayTokenSymbolOrLink(trade.sellToken) + '/' + displayTokenSymbolOrLink(trade.buyToken),
```

Comment:
I think in the CSV we should display both columns. Symbol (possibly empty) and address. More data wouldn't hurt.